pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/FormResponse.java

```diff
@@ -36,9 +36,12 @@ public FormResponse(@ColumnName("instance_id") long id,
                         @ColumnName("activity_id") long activityId,
                         @ColumnName("activity_code") String activityCode,
                         @ColumnName("activity_version_tag") String activityVersionTag,
+                        @ColumnName("is_hidden") Boolean isHidden,
+                        @ColumnName("sectionIndex") Integer sectionIndex,
```

Comment:
JDBI is smart enough to work with both snake_case and camelCase, but let's be consistent here.
```suggestion
                        @ColumnName("section_index") Integer sectionIndex,
```